src/OSPSuite.Presentation/Presenters/Charts/CommonAnalysisChartPresenter.cs

```diff
@@ -0,0 +1,130 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using OSPSuite.Core.Chart;
+using OSPSuite.Core.Domain;
+using OSPSuite.Core.Domain.Data;
+using OSPSuite.Core.Domain.ParameterIdentifications;
+using OSPSuite.Core.Extensions;
+using OSPSuite.Presentation.Extensions;
+using OSPSuite.Presentation.Services.Charts;
+using OSPSuite.Presentation.Views;
+using OSPSuite.Utility.Collections;
+using OSPSuite.Utility.Extensions;
+
+namespace OSPSuite.Presentation.Presenters.Charts
+{
+   public interface ICommonAnalysisChartPresenter : ISimulationAnalysisPresenter
+   {
+   }
+
+   public abstract class CommonAnalysisChartPresenter<TChart, TView, TPresenter> : SimulationAnalysisChartPresenter<TChart, TView, TPresenter>,
```

Comment:
why does common means? What is the difference with SimulationAnalysisChartPresenter
Name is so what we know what is implemented

Reply:
Common here refers to both PI and Simulation. But I agree this is not optimal: 

The thing is there exists a SimulationAnalysisChartPresenter that is also implemented by ParameterIdentificationAnalysisChartPresenter. I would actually suggest to merge these two presenters into one, then we could call it AnalysisChartPresenter?

Reply:
I am a bit confused now. I still don't understand what this Common presenter is about. What presenter implement this? What is the difference with SimulationAnalysisChartPresenter?